cdap-app-fabric/src/test/java/io/cdap/cdap/config/SqlUserConfigStoreTest.java

```diff
@@ -49,11 +56,21 @@ public static void setup() throws Exception {
     cConf.set(Constants.REQUIREMENTS_DATASET_TYPE_EXCLUDE, Joiner.on(",").join(Table.TYPE, KeyValueTable.TYPE));
 
     pg = PostgresInstantiator.createAndStart(cConf, TEMP_FOLDER.newFolder());
-    DataSource dataSource = pg.getPostgresDatabase();
-    SqlStructuredTableRegistry registry = new SqlStructuredTableRegistry(dataSource);
-    registry.initialize();
-    admin = new PostgresSqlStructuredTableAdmin(registry, dataSource);
-    TransactionRunner transactionRunner = new SqlTransactionRunner(admin, dataSource);
+    Injector injector = Guice.createInjector(
```

Comment:
This looks same for basically all sql related table unit tests, can this be refactored into a common place?

Reply:
Currently these tests are scattered across different modules, making it kind of hard to refactor. Ideally we should move the postgreSQL implementation as an extension, and have tests defined there.

Open https://cdap.atlassian.net/browse/CDAP-18629 as a followup task.